wdio.conf.js

```diff
@@ -183,7 +184,17 @@ exports.config = {
     services: [
         ['sauce', {
             sauceConnect: true
-        }]
+        }],
+        ['image-comparison',
+            // The options
+            {
+                // Some options, see the docs for more
+                baselineFolder: join(process.cwd(), './e2e/wdio/platform/baselineScreenshot/'),
+                formatImageName: '{tag}-{logName}-{width}x{height}',
+                screenshotPath: join(process.cwd(), '.tmp/'),
+                savePerInstance: true,
+                autoSaveBaseline: true,
+            }],
```

Comment:
Why introduce this here when you will raise another PR today with this service and implemented checks?